### DISCUSSION TITLE
文档可能需要更新

### DISCUSSION THREAD

**Discussion by batu1579** [CaiJimmy/hugo-theme-stack#637] [General]
文档中对于组件的描述和当前的版本好像不太匹配

文档中提到了`enabled` 、 `archives` 、 tagCloud``三个字段，但是在示例当中使用的是 `homepage` 和 `page` 。也不是通过数组的形式存储的。

希望大大有时间可以更新一下文档

**Comment by CaiJimmy** [comment]
抱歉，最近没有时间。有空会试着去更新的。
